src/features/analytics/index.ts

```diff
@@ -2,7 +2,7 @@ if (process.env.NODE_ENV === 'production' && !process.env.NEXT_PUBLIC_GA_ID) {
   throw new Error('Missing NEXT_PUBLIC_GA_ID')
 }
 
-export const GA_TRACKING_ID = process.env.NEXT_PUBLIC_GA_ID
+export const GA_TRACKING_ID = process.env.NEXT_PUBLIC_GA_ID || ''
```

Comment:
Yes, otherwise it fails linting, picked up during testing of the build process. 

This change ensures that `GA_TRACKING_ID` to is always a `string` rather than `string | undefined`.